### DISCUSSION TITLE
Where are all my 10000 plots? (1000Tb / 1Pb)

### DISCUSSION THREAD

**Discussion by Kryptoand** [Chia-Network/chia-blockchain#7304] [Support]
Hello everybody! We have been making plots for 2 months. Since we have installed the new version 1.2 when we open the program there is not a single plot. We have added a pool like space.pool to see if this was the problem but no. The 10000 plots that we have been doing during these 2 months do not work? They said there would be no re-plots... someone can help me? Thanks!

**Comment by joelrb** [comment]
Known Issues
Sometimes plots don't show up instantly when starting the GUI. Click the refresh button in the Plots page.

From release notes:
https://github.com/Chia-Network/chia-blockchain/releases

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
